opensalt/opensalt
# Advance browser to new framework on clone

Original issue:
https://github.com/opensalt/opensalt/issues/883

**Test plan (required)**

1. Login to OpenSALT
2. Open an unpublished framework
3. Click the "clone framework" button
4. Browser should advance to the newly-created framework and open the Edit Framework modal



**Code formatting Notes**

Notes:
- Use Spaces not tabs


**Closing issues**

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/opensalt/opensalt/884)
